docs/content/preview/releases/release-notes/v2.15.md

```diff
@@ -274,7 +274,7 @@ docker pull yugabytedb/yugabyte:2.15.3.0-b231
 
 #### YugabyteDB Anywhere
 
-* [PLAT-465] [PLAT-3501] Changing volume size of kubernetes universe results in error, Copy functionality is enabled when empty
+* [PLAT-465] [PLAT-3501] Changing volume size of Kubernetes universe results in error, Copy functionality is enabled when empty
```

Comment:
line 243 - change GUC variables

Reply:
Yes I left the references in the RN and API as special cases. API may change with another PR